### PR TITLE
Redesing settings on Android

### PR DESCRIPTION
**Pull request by osdnk**
image

### CODE REVIEW

File: src/components/modal/Modal.js [hunk 1]
```diff
@@ -45,7 +46,11 @@ export default function Modal({
       <Content
         {...props}
         fixedToTop={fixedToTop}
-        height={height || deviceHeight - 220}
+        height={
+          (fullScreenOnAndroid && Platform.OS === 'android'
```
Comment (brunobar79) [inline src/components/modal/Modal.js:50]: if you merge the other PR first you might wanna use the global here

File: src/screens/SettingsModal.js [hunk 2]
```diff
@@ -171,9 +174,11 @@ export default function SettingsModal() {
 
   return (
     <Modal
+      fullScreenOnAndroid
       minHeight={isTinyPhone ? 500 : 600}
       onCloseModal={goBack}
       radius={18}
+      showDoneButton={Platform.OS === 'ios'}
```
Comment (brunobar79) [inline src/screens/SettingsModal.js:181]: same

File: src/screens/SettingsModal.js [hunk 3]
```diff
@@ -203,7 +208,6 @@ export default function SettingsModal() {
               height: 49,
               shadowColor: 'transparent',
             },
-            headerTitleAlign: 'center',
```
Comment (brunobar79) [inline src/screens/SettingsModal.js:206]: Wouldn't this affect iOS?

**Review by brunobar79 — APPROVED** [review]

File: src/screens/SettingsModal.js (hunk 3 above)
Comment (osdnk) [reply src/screens/SettingsModal.js:206]: no, it defaults to 'center' on iOS

**Pull request merged**